styleguide/components/11-base/fonts/_fonts.scss

```diff
@@ -1 +1 @@
-@import url('https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700,400italic'); // sass-lint:disable-line no-url-protocols
+@import url('https://fonts.googleapis.com/css?family=Fira+Sans'); // sass-lint:disable-line no-url-protocols
```

Comment:
Let's only import the font variants that are applicable and supported in styleguide :)

Reply:
Isn't that what we're doing here? :)